.github/workflows/Func_Env_Build_Test_CI.yml

```diff
@@ -38,14 +38,16 @@ jobs:
         flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
         # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
         flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
-    - name: ⚙️ Set Kubeconfig
+    - name: ⚙️ Set Kubeconfig and hosts
       env:
         KUBECONFIG: ${{ secrets.FUNC_KUBECONFIG }}
         RUNNER_PATH: ${{ secrets.RUNNER_PATH }}
+        FUNC_OCP_HOSTS: ${{ secrets.FUNC_OCP_HOSTS }}
       run: |
           mkdir -p "$RUNNER_PATH/.kube/"
           echo "$KUBECONFIG" > "$RUNNER_PATH/.kube/config"
           echo "KUBECONFIG_PATH=$RUNNER_PATH/.kube/config" >> "$GITHUB_ENV"
+          sudo echo "$FUNC_OCP_HOSTS" | sudo tee -a "/etc/hosts"
```

Comment:
```
sudo tee -a /etc/hosts <<< "$FUNC_OCP_HOSTS" >/dev/null
```
You don't want the `tee -a` output going to stdout where it might be seen in logs.  Same applies to all of the other uses of this construct.

Reply:
Fixed